packages/@sparrow-workspaces/src/hoc/WithModal.svelte

```diff
@@ -26,9 +26,10 @@
       file?
     </small>
   </div>
-  <div class="d-flex justify-content-between">
-    <div style="margin-right: 15px;">
+  <div class="row">
+    <div class="col-12 mb-2 col-xl-5 col-xxl-3">
```

Comment:
layout is changing on smaller screens. layout needs to be persistent. all the 3 buttons in a row. may be we can increase modal width.